scripts/remove_after_use/reindex_quickfiles.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+import progressbar
+from django.core.paginator import Paginator
+
+from website.app import setup_django
+setup_django()
+
+from website.search.search import update_file
+from osf.models import QuickFilesNode
+
+PAGE_SIZE = 50
+
+def reindex_quickfiles(dry):
+    qs = QuickFilesNode.objects.all().order_by('id')
```

Comment:
How about iterating over files that are attached to QF nodes instead? 
```
OsfStorageFile.objects.filter(node__type='osf.quickfilesnode')
```
No need to check `quickfiles.files.all()` when it's (probably) empty for most QF nodes.